pact.gemspec

```diff
@@ -38,7 +37,7 @@ Gem::Specification.new do |gem|
 
   gem.add_development_dependency 'rake', '~> 10.0.3'
   gem.add_development_dependency 'webmock', '~> 3.0'
-  #gem.add_development_dependency 'pry-byebug'
```

Comment:
Perhaps we can remove this dependency in the gemspec instead, given https://github.com/pact-foundation/pact-ruby/blob/master/Gemfile#L9?

Reply:
Nah, sometimes debugging is useful even when not using the local gems.